index.js

```diff
@@ -9,11 +9,12 @@ module.exports = filename => {
 	const pkg = require(path.join(globalDir, 'package.json'));
 	const localFile = resolveCwd.silent(path.join(pkg.name, relativePath));
 	const localNodeModules = path.join(process.cwd(), 'node_modules');
-	const filenameInLocalNodeModules = !path.relative(localNodeModules, filename).startsWith('..');
+	const fileInLocalPath = path.relative(localNodeModules, filename);
+	const filenameInLocalNodeModules = fileInLocalPath.startsWith('..') || (os.type() === 'Windows_NT' && path.isAbsolute(fileInLocalPath));
```

Comment:
This is not the correct way to check for Windows.

Ideally, the solution would be something that works on both platforms without a platform specific check.